src/plugins/workspace/public/components/workspace_detail/workspace_detail.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiFlexItem,
+  EuiTabbedContent,
+  EuiFlexGroup,
+  EuiPanel,
+} from '@elastic/eui';
+
+import { useObservable } from 'react-use';
+import { i18n } from '@osd/i18n';
+import { CoreStart, PublicAppInfo } from 'opensearch-dashboards/public';
+import { BehaviorSubject } from 'rxjs';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+import { WorkspaceDetailContent } from './workspace_detail_content';
+import { WorkspaceUpdater } from './workspace_updater';
+import { DetailTab } from '../workspace_form/constants';
+
+export interface WorkspaceDetailProps {
+  workspaceConfigurableApps$?: BehaviorSubject<PublicAppInfo[]>;
```

Comment:
I had an impression that @wanglam removed `workspaceConfigurableApps$`, @wanglam can you have a look at this?